src/components/measureLanding/measureList/MeasureList.test.tsx

```diff
@@ -1344,6 +1554,10 @@ describe("Measure List component", () => {
     });
     await waitFor(() => {
       expect(getByText("Measure exported successfully")).toBeInTheDocument();
+      const continueButton = getByTestId("ds-btn");
+      expect(continueButton).toBeInTheDocument();
+      fireEvent.click(continueButton);
+      expect(queryByText("Measure exported successfully")).toBeInTheDocument();
```

Comment:
There is already a check to see if this message is available even before continueButton is clicked ?

Reply:
I guess I am concerned of how did the tests pass successfully ?

Reply:
@RohitKandimalla the toast message was there, so it would pass.

Reply:
So the toast message will appear before and after a user clicks on continue button ?